espresso/environment/espresso_dev_node_test.go

```diff
@@ -127,6 +128,7 @@ func runSimpleL1TransferAndVerifier(ctx context.Context, t *testing.T, system *e
 			t.Errorf("waiting for balance change returned with error:\nhave:\n\t\"%v\"\nwant:\t\n\"%v\"\n", have, want)
 		}
 
+		log.Println("endBalance:", endBalance, "startBalance:", startBalance)
```

Comment:
A lot of extraneous logging here can be removed, used for debugging

Reply:
Follow-up task: https://app.asana.com/1/1208976916964769/project/1209392461754458/task/1209980878478912?focus=true